Makefile

```diff
@@ -5,24 +5,21 @@ else
 	ENVBIN="bin"
 endif
 
-BASE := $(shell git merge-base HEAD origin/master)
-
 env:
-	virtualenv env
+	python -m virtualenv env
 
-requirements: env
-	. env/$(ENVBIN)/activate && pip install -r requirements.txt
+requirements:
+	pip install -r requirements.txt
 
-lint: requirements
-	. env/$(ENVBIN)/activate && pylint *.py
-	. env/$(ENVBIN)/activate && mypy .
+lint:
+	pylint *.py
+	mypy .
 
-test: requirements
-	. env/$(ENVBIN)/activate && flit install
-	. env/$(ENVBIN)/activate && pytest -v --cov=. --cov-report=xml --durations=1 -n=2 --provider=onedrive,testodbiz tests
+test:
+	pytest -v --cov=. --cov-report=xml --durations=1 -n=2 --provider=onedrive,testodbiz tests
 
 format:
 	autopep8 --in-place -r -j 8 cloudsync/
 
 coverage:
-	diff-cover coverage.xml --compare-branch=$(BASE)
+	diff-cover coverage.xml --compare-branch=$(shell git merge-base HEAD origin/master)
```

Comment:
what is shell?

Reply:
the `make` way of fetching the result of a shell call, AFAICT.

this was moved from line 8 in Makefile, which was generating an error for some reason.